admin_notifications/helpers/create_notification.py

```diff
@@ -0,0 +1,34 @@
+from admin_notifications.models import AdminNotification
+from api.location.models import Location
+from datetime import datetime
+import celery
+
+
+def update_notification(notification_id):
+    notification = AdminNotification.query.filter_by(id=notification_id).first()
+    notification.date_received = datetime.now()
+    notification.save()
+
+
+@celery.task
```

Comment:
Please specify the name of the task that is run asynchronously. Such as `@celery.task(name='asynchronous-email-notifications')
`